app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/wiki/selector/WikiSelectorScreen.kt

```diff
@@ -0,0 +1,217 @@
+package io.eugenethedev.taigamobile.ui.screens.wiki.selector
```

Comment:
I think it's better to name it `list` instead of `selector`. And, respectively, rename composables and classes from `WikiSelector...` to `WikiList...`